### PR TITLE
M sc1 61 19

### PR DESCRIPTION
**Pull request by sergey-mario**
This is my hometask "CSS 04".
But I can't understand how I can change align at right column using "not" (file "work_table"). That's why I did task whithout "not".

### CODE REVIEW

**Comment by pluhin** [comment]
> This is my hometask "CSS 04".
> But I can't understand how I can change align at right column using "not" (file "work_table"). That's why I did task whithout "not".
```
tr:not(:nth-child(1)) th { text-align: left;}
```

File: smaretski/04_CSS/chess_board_num.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    }
+    tr, td {border: 1px solid black;
+    width: 100px;
+    height: 100px;
+    text-align: center;
+    font-style: italic;
+    font-size: 50px;
+    
+    }
+    tr:nth-child(odd) > td:nth-child(even){
```
Comment (pluhin) [inline smaretski/04_CSS/chess_board_num.html]: You can combine with next one

File: smaretski/04_CSS/chess_board.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    table {border: 2px solid black;
+    border-collapse: collapse;
+    outline: 5px solid black;
+    margin: 20px 20px 20px 20px;
+    }
+    tr, td {border: 1px solid black;
+    width: 100px;
+    height: 100px;
+    }
+    tr:nth-child(odd) > td:nth-child(even){
```
Comment (pluhin) [inline smaretski/04_CSS/chess_board.html]: you can combine with next one

**Review by pluhin — CHANGES_REQUESTED** [review]

File: smaretski/04_CSS/chess_board_num.html (hunk 1 above)
Comment (sergey-mario) [reply smaretski/04_CSS/chess_board_num.html]: Combined

File: smaretski/04_CSS/chess_board.html (hunk 2 above)
Comment (sergey-mario) [reply smaretski/04_CSS/chess_board.html]: Combined

**Pull request merged**